src/main/js/controls/Scatter/Scatter.js

```diff
@@ -184,6 +188,36 @@ class Scatter extends GraphContent {
         return this;
     }
 
+    update(graph, graphData) {
```

Comment:
Why not just call this `reflow` as well? Following the same pattern as `resize`, both `Construct` and `GraphContent` have the same interfaces. 
I would also add this to the `Construct` and `GraphContent` interfaces.